core/src/test/java/io/substrait/type/proto/JoinRoundtripTest.java

```diff
@@ -41,10 +43,13 @@ public class JoinRoundtripTest extends TestBase {
           Arrays.asList("d", "e", "f"),
           Arrays.asList(R.FP64, R.STRING, R.I64));
 
-  void verifyRoundTrip(Rel rel) {
+  Rel roundTrip(Rel rel) {
     io.substrait.proto.Rel protoRel = relProtoConverter.toProto(rel);
-    Rel relReturned = protoRelConverter.from(protoRel);
-    assertEquals(rel, relReturned);
+    return protoRelConverter.from(protoRel);
+  }
```

Comment:
Looks better. :+1:

Reply:
Thanks! I split up roundtrip functionality from verification-of-roundtrip so I could test inequality for NLJ more easily.